zebra-network/src/constants.rs

```diff
@@ -43,7 +43,19 @@ pub const HANDSHAKE_TIMEOUT: Duration = Duration::from_secs(4);
 /// This avoids explicit synchronization, but relies on the peer
 /// connector actually setting up channels and these heartbeats in a
 /// specific manner that matches up with this math.
-pub const LIVE_PEER_DURATION: Duration = Duration::from_secs(60 + 20 + 20 + 20);
+pub const MIN_PEER_RECONNECTION_DELAY: Duration = Duration::from_secs(60 + 20 + 20 + 20);
+
+/// The maximum duration since a peer was last seen to consider it reachable.
+///
+/// This is used to prevent Zebra from gossiping networks that are likely unreachable. Peers that
```

Comment:
Typo - I'll just fix then merge
```suggestion
/// This is used to prevent Zebra from gossiping addresses that are likely unreachable. Peers that
```

Reply:
Oops. Thanks for catching that 😁